### PR TITLE
Added WDQS usage script & dashboard

### PR DESCRIPTION
**Pull request by bearloga**

### CODE REVIEW

File: assets/content/wdqs_basic.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+Wikidata Query Service (WDQS) usage
+=======
+
```
Comment (Ironholds) [inline assets/content/wdqs_basic.md]: Waitwitwait. We're storing WDQS in the same dashboard?

File: data_retrieval/wdqs.R [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+library(olivr)
+suppressPackageStartupMessages(library(data.table))
+
+if(!file.exists(base_path)){
+  dir.create(path = base_path)
+}
+
+#Conditional write; if the file exists, add x to the end. If it doesn't, write an entirely new file.
+conditional_write <- function(x, file) {
+  if(file.exists(file)){
```
Comment (Ironholds) [inline data_retrieval/wdqs.R]: This already exists in a utils file, no?

File: data_retrieval/wdqs.R [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+  # Write query and dump to file
+  query <- paste0("USE wmf;
+                   SELECT year, month, day,
+                   FIND_IN_SET(uri_path, '/bigdata/namespace/wdq/sparql,/,/index.php') AS uri_path,
+                   IF(INSTR(uri_query, 'query') > 0, 'query', 'other') AS uri_query,
+                   IF(INSTR(content_type, 'sparql-results') > 0, 'sparql results', 'other') AS content_type,
+                   COUNT(*) AS n
+                   FROM webrequest",
+                   subquery,
+                  "AND webrequest_source = 'misc' AND FIND_IN_SET(http_status, '200,301,302,303') > 0
```
Comment (Ironholds) [inline data_retrieval/wdqs.R]: Why do you need FIND IN SET? What's the difference between this and http_status IN('200','301','302','303')?

File: server.R [hunk 4]
```diff
@@ -53,6 +53,14 @@ read_api <- function(){
   return(invisible())
 }
 
+read_wdqs <- function(){
+  con <- url("http://datasets.wikimedia.org/aggregate-datasets/wdqs/wdqs_aggregates.tsv")
+  data <- readr::read_delim(con, delim = "\t")
```
Comment (Ironholds) [inline server.R]: dude check out download_set()

File: assets/content/wdqs_basic.md (hunk 1 above)
Comment (bearloga) [inline assets/content/wdqs_basic.md]: Oh. That's not what I was supposed to do? Okay.

The point of confusion was that everyone refers to the individual modules within a dashboard as dashboards.

File: data_retrieval/wdqs.R (hunk 3 above)
Comment (bearloga) [inline data_retrieval/wdqs.R]: No idea. IN() may be faster. Will change accordingly.

**Pull request closed without merging**